2/2simple_task_c++/dE_dt_plot.py

```diff
@@ -0,0 +1,17 @@
+import matplotlib.pyplot as plt
+import re
+
+f = open('dE_dt.txt', 'r')
+E_list = []
+dt_list = []
+
+for line in f:
```

Comment:
Yet again why did you read your text files like this? It's much more convenient to read them by means of np.loadtxt. No need to write custom parser with regexps.